src/main/resources/application.yml

```
@@ -135,10 +135,22 @@ rest:
     wait-time-millis: "\${PROCESS_EXECUTIONS_WAIT_TIME_MILLIS:500}"
     print-body-when-error: "\${PROCESS_EXECUTIONS_PRINT_BODY_WHEN_ERROR:true}"
   fileshare:
-    base-url: "\${FILESHARE_BASE_URL:}"
+    base-url: "\${FILE_SHARE_BASE_URL:}"
     max-attempts: "\${FILE_SHARE_MAX_ATTEMPTS:3}" # It's counting also the first attempt
     wait-time-millis: "\${FILE_SHARE_WAIT_TIME_MILLIS:500}"
     print-body-when-error: "\${FILE_SHARE_PRINT_BODY_WHEN_ERROR:true}"
+  organization:
+    base-url: "\${ORGANIZATION_BASE_URL:http://localhost:8080/organization}"
```

this default could lead to a misunderstanding, you are not expecting to have a basePath.
don't set neither a value or let's not set a different basePath
```suggestion
    base-url: "\${ORGANIZATION_BASE_URL:http://localhost:8080}"
```